tests/test_arraycreation.py

```diff
@@ -258,48 +258,44 @@ def test_tri_default_dtype():
 
 
 @pytest.mark.parametrize("k",
-                         [-6, -5, -4, -3, -2, -1, 0, 1, 2, 3, 4, 5, 6],
-                         ids=['-6', '-5', '-4', '-3', '-2', '-1', '0', '1', '2', '3', '4', '5', '6'])
+                         [-3, -2, -1, 0, 1, 2, 3, 4, 5],
```

Comment:
Can we add here (and in test below) 0-dimension array of dpnp_array and usm_ndarray and numpy scalar?